update lightning faq (#4574)
pingcap/docs: tidb-lightning/tidb-lightning-backends.md

### Migrating from Loader to TiDB Lightning TiDB-backend

> When you need to import data into the TiDB clusters, TiDB Lightning using the TiDB-backend can completely replace functions of [Loader](https://docs.pingcap.com/tidb/v4.0/loader-overview). The following list shows how to translate Loader configurations into [TiDB Lightning configurations](/tidb-lightning/tidb-lightning-configuration.md).

Reviewer: ```suggestion
If you need to import data into a TiDB cluster, TiDB Lightning using the TiDB-backend can completely replace the functionalities of [Loader](https://docs.pingcap.com/tidb/v4.0/loader-overview). The following list shows how to translate Loader configurations into [TiDB Lightning configurations](/tidb-lightning/tidb-lightning-configuration.md).
```
